src/adapters/serverbid.js

```diff
@@ -44,43 +48,67 @@ const ServerBidAdapter = function ServerBidAdapter() {
 
   baseAdapter.callBids = function(params) {
     if (params && params.bids && utils.isArray(params.bids) && params.bids.length) {
-      const data = {
-        placements: [],
-        time: Date.now(),
-        user: {},
-        url: utils.getTopWindowUrl(),
-        referrer: document.referrer,
-        enableBotFiltering: true,
-        includePricingData: true
+      window[SMARTSYNC_CALLBACK] = function() {
+        _callBids(params);
```

Comment:
Talked about it and decided that optional is ok as long as the publisher has to opt in.  
BTW -  the draft proposal for a 1.0 release is to force sync urls to be registered with prebid and then run later, after some timeout.  They'd also be restricted to pixels, iframes, or xhr (no jsonp for security reasons).   We'll have to think about your use case.  this will be put out for review at some point in the near future.